projects/angular/src/layout/nav/_header.clarity.scss

```
@@ -588,95 +344,48 @@
-          clr-header-search-icon-size,
-          header-variables.$clr-search-icon-width,
-          variables.$clr-use-custom-properties
-        );
-        margin: variables.$clr_baselineRem_0_83 0 0;
-        vertical-align: top;
-        @include mixins.css-var(
-          margin-right,
-          clr-header-search-icon-margin-right,
-          0,
-          variables.$clr-use-custom-properties
+        @include mixins.min-equilateral(header-variables.$clr-header-search-icon-size);
+        vertical-align: middle;
+        margin: 0;
+        margin-right: calc(
+          -1 * (header-variables.$clr-header-search-icon-size + header-variables.$clr-header-search-gap)
         );
       }
 
       input {
-        line-height: variables.$clr_baselineRem_1;
-        @include mixins.css-var(
-          margin,
-          clr-header-search-margin,
-          header-variables.$clr-header-nav-text-vertical-padding 0,
-          variables.$clr-use-custom-properties
-        );
+        line-height: tokens.$cds-global-space-9;
+        margin: auto;
       }
     }
 
     input[type='text'] {
       border: none;
       background: none;
-      @include mixins.css-var(
-        color,
-        clr-header-search-input-font-color,
-        header-variables.$clr-header-font-color,
-        variables.$clr-use-custom-properties
-      );
-
-      @include mixins.css-var(
-        height,
-        clr-header-search-height,
-        variables.$clr_baselineRem_1,
-        variables.$clr-use-custom-properties
-      );
+      color: header-variables.$clr-header-search-input-font-color;
+
+      height: header-variables.$clr-header-search-height;
 
       padding: 0;
-      @include mixins.css-var(padding-left, clr-header-search-input-padding, 0, variables.$clr-use-custom-properties);
+      padding-left: calc(header-variables.$clr-header-search-icon-size + 2 * header-variables.$clr-header-search-gap);
 
       vertical-align: middle;
 
       // Typography
-      @include mixins.css-var(
-        font-size,
-        clr-header-search-input-font-size,
-        inherit,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-weight,
-        clr-header-search-input-font-weight,
-        inherit,
-        variables.$clr-use-custom-properties
-      );
+      font-size: header-variables.$clr-header-search-input-font-size;
+      font-weight: header-variables.$clr-header-search-input-font-weight;
+
       &::placeholder {
-        @include mixins.css-var(
-          font-weight,
-          clr-header-search-placeholder-font-weight,
-          inherit,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          color,
-          clr-header-search-placeholder-text-color,
-          forms-variables.$clr-forms-placeholder-color,
-          variables.$clr-use-custom-properties
-        );
+        font-weight: header-variables.$clr-header-search-placeholder-font-weight;
```

`header-variables.$clr-header-search-placeholder-font-weight` is redundant.

```suggestion
        @include mixins.generate-typography-token('BODY-14-RG-EXP');
```

---

Also removed from variables, properties and STYLES